identity/app/form/Mappings.scala

```diff
@@ -22,7 +22,8 @@ trait Mappings {
     { value => value.isEmpty || UrlPattern.findFirstIn(value).isDefined }
   )
 
-  private val EmailPattern = """(\w+)@([\w\.]+)""".r
```

Comment:
Oops ... 

. will also match spaces. So this will consider "      @      " an e-mail address, as well as just "@". If that doesn't matter, cool.